redfish_utilities/inventory.py

```diff
@@ -284,3 +286,49 @@ def print_system_inventory( inventory_list, details = False, skip_absent = False
                             if item[detail] is not None:
                                 print( inventory_line_format_detail.format( "", detail, item[detail] ) )
         print( "" )
+
+
+def write_system_inventory( inventory_list):
+    """
+    Write the system inventory list into a spreadsheet
+
+    Args:
+        inventory_list: The inventory list to write to an Excel spreadsheet
+    """
+
+    # Excel workbook to save data extracted and parsed
+    workbook = xlsxwriter.Workbook("./Device_Inventory.xlsx")
```

Comment:
Could we make this an argument for the function (and have it default to "./Device_Inventory.xlsx" if not provided)?

Reply:
That should be possible. I'll see what I can come up with.

Reply:
I have updated the pull request. Is what I added what you had in mind?